src/nxt_php_sapi.c

```diff
@@ -65,6 +65,15 @@ typedef struct {
 } nxt_php_run_ctx_t;
 
 
+HashTable nxt_php_user_config_cache;
+
+
+typedef struct {
+    nxt_time_t  expires;
+    HashTable   *user_config;
+} user_config_cache_entry;
```

Comment:
According to our style, type name should be prefixed by `nxt_php_` and ends with `_t`. So, the correct type name here is `nxt_php_ user_config_cache_entry_t`.